chronos/__init__.py

```diff
@@ -26,13 +26,16 @@
 import socket
 import json
 import logging
+import warnings
 
 # Python 3 changed the submodule for quote
 try:
     from urllib import quote
 except ImportError:
     from urllib.parse import quote
 
+SCHEDULER_VERSIONS = ('v1',)
```

Comment:
can we rename this (and all mentions of ``scheduler_version`` to be ``API_VERSION`` instead? it just makes it a little clearer what this actually does.